lib/modules/apostrophe-assets/public/css/site.less

```diff
@@ -131,6 +131,17 @@ h4 p {
   border-width: 1px;
 }
 
+.search {
+  background-image: url(../images/magnifying_glass.png);
+  background-repeat: no-repeat;
+  cursor: pointer;
+  background-position: 0px 0px;
+  border: 0px;
+  height: 4rem;
+  width: 4rem;
+  background-size: cover;
```

Comment:
I might have looked to instead use tachyon classes (`bg-center pointer h3 w3`) instead of `.search` for some of these, but this way is quite reusable 👍